frontend/lib/components/ui/ams/AmsStatus.svelte

```diff
@@ -0,0 +1,189 @@
+                                                    ></span>
+                                                </div>
+                                                <div>
+                                                    <span class="font-semibold">Nozzle Temperature:</span>
+                                                    <div class="ml-2">
+                                                        <span>max: {slot.nozzle_temp_max}°C</span>
+                                                        <span class="ml-4">min: {slot.nozzle_temp_min}°C</span>
+                                                    </div>
+                                                </div>
+                                                <div>
+                                                    <span class="font-semibold">Serial Number:</span>
+                                                    <span>{slot.tag_uid}</span>
+                                                </div>
+
+                                                <div class="mt-3 font-semibold">Flow Dynamics</div>
+                                                <div class="ml-2">
+                                                    <div>
+                                                        <span class="font-semibold">Factor K:</span>
+                                                        <span>{slot.k_factor}</span>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        <Popover.Arrow class={`${tailwindTextColorClass(formatColor(slot.color))}`} />
+                                        </Popover.Content>
+                                    </Popover.Portal>
+                                </Popover.Root>
+                            {:else}
+                                <span class="text-slate-400">?</span>
+                            {/if}
+                        </div>
+                        {#if slot.active}
+                            <div class="absolute -bottom-10 flex w-full flex-col items-center">
+                                <div class="h-5 w-0.5 bg-slate-600"></div>
+                                <div class="-mt-px flex h-5 w-5 items-center justify-center rounded-full bg-slate-700">
+                                    <div
+                                        class="h-2.5 w-2.5 rounded-full"
+                                        style="background-color: {formatColor(slot.color)}"
+                                    ></div>
+                                </div>
+                            </div>
+                        {/if}
+                    </div>
+                {/each}
+            </div>
+        </div>
+    </div>
+
+    <div class="mt-8 flex gap-3">
+        <Button variant="outline" class="w-16" disabled>Unload</Button>
```

Comment:
cant be some buttons be connected to the same commands as the current load/unload?